src/sas/sascalc/dataloader/data_info.py

```diff
@@ -37,11 +41,11 @@ class plottable_1D(object):
     y = None
     dx = None
     dy = None
-    ## Slit smearing length
+    # Slit smearing length
```

Comment:
Autodoc can recognize lines starting with #: as docstrings for the class attributes that follow.

See for example the ModelInfo [definition](http://www.sasview.org/docs/_modules/sasmodels/modelinfo.html#ModelInfo) and [docs](http://www.sasview.org/docs/dev/sasmodels-api/sasmodels.html#model-info-and-parameter-tables).

Reply:
That's new to me! I'll add this to another of my upcoming updates.